### ISSUE TITLE
Example fails with numba error 

### ISSUE THREAD

**Issue by quazzuk** [marcdemers/py_vollib_vectorized#2]
Hi 

If I install and run the code snippet in the Readme I get:

````
---------------------------------------------------------------------------
TypingError                               Traceback (most recent call last)
<ipython-input-4-041cde1638d1> in <module>
     12 iv = 0.2  # Implied Volatilities
     13 
---> 14 option_price = py_vollib.black_scholes.black_scholes(flag, S, K, t, r, iv, return_as='array')
     15 # array([12.11158143,  2.02418536])

~/miniconda3/envs/dev/lib/python3.8/site-packages/py_vollib_vectorized/models.py in vectorized_black_scholes(flag, S, K, t, r, sigma, return_as, dtype)
     76     _validate_data(S, K, sigma, t, r, flag)
     77 
---> 78     prices = _black_scholes_vectorized_call(flag, S, K, t, r, sigma)
     79 
     80     if return_as == "series":

~/miniconda3/envs/dev/lib/python3.8/site-packages/numba/core/dispatcher.py in _compile_for_args(self, *args, **kws)
    412                 e.patch_message(msg)
    413 
--> 414             error_rewrite(e, 'typing')
    415         except errors.UnsupportedError as e:
    416             # Something unsupported is present in the user code, add help info

~/miniconda3/envs/dev/lib/python3.8/site-packages/numba/core/dispatcher.py in error_rewrite(e, issue_type)
    355                 raise e
    356             else:
--> 357                 raise e.with_traceback(None)
    358 
    359         argtypes = []

TypingError: Failed in nopython mode pipeline (step: nopython frontend)
No implementation of function Function(<intrinsic range_iter_len>) found for signature:
 
<unknown function>(zip(iter(array(float64, 1d, C)), iter(array(float64, 1d, C)), iter(array(float64, 1d, C)), iter(array(float64, 1d, C)), iter(array(float64, 1d, C)), iter(array(float64, 1d, C))))
 
There are 2 candidate implementations:
   - Of which 2 did not match due to:
   Intrinsic of function '<unknown function>': File: numba/cpython/rangeobj.py: Line 183.
     With argument(s): '(zip(iter(array(float64, 1d, C)), iter(array(float64, 1d, C)), iter(array(float64, 1d, C)), iter(array(float64, 1d, C)), iter(array(float64, 1d, C)), iter(array(float64, 1d, C))))':
    No match.

During: resolving callee type: Function(<intrinsic range_iter_len>)
During: typing of call at /home/andrewdouglas/miniconda3/envs/dev/lib/python3.8/site-packages/py_vollib_vectorized/_model_calls.py (83)


File "../../../../../miniconda3/envs/dev/lib/python3.8/site-packages/py_vollib_vectorized/_model_calls.py", line 83:
def _black_scholes_vectorized_call(flags, Ss, Ks, ts, rs, sigmas):
    prices = []
    ^
```

py_vollib_vectorized = 0.1 and numba = 0.52

Any ideas what's causing this?

Thanks
Andy

**Comment by quazzuk** [comment]
It's seems to be related to `zip`. Numba does say it supports `zip` but perhaps the implementation is broken for my arch/os (`numba-0.52.0-cp38-cp38-manylinux2014_x86_64.whl`). Swapping out to use `for i in range(len(...))` and indexing the using i seems to workaround the issue but is a pain as the zip unpacking pattern is used pretty heavily. Are you running on linux or a different os?

**Comment by marcdemers** [comment]
Numba supports zip [out-of-the-box](https://numba.pydata.org/numba-doc/dev/reference/pysupported.html?highlight=zip).

Could you try downgrading to Py3.7 or Numba 0.47 to see if that makes a difference and report back?

**Comment by marcdemers** [comment]
@quazzuk  FYI I found the issue, should be solved in 0.0.1

Thanks!
